{
  "repository": {
    "name": "hekate-io/hekate",
    "githubId": 84603051,
    "url": "https://github.com/hekate-io/hekate",
    "isFork": false,
    "original": {
      "name": "",
      "githubId": 0,
      "url": ""
    },
    "isPullRequest": false,
    "pullRequestId": 0
  },
  "builds": {
    "buggyBuild": {
      "id": 438091780,
      "url": "http://travis-ci.org/hekate-io/hekate/builds/438091780",
      "date": "Oct 6, 2018 10:33:13 PM"
    }
  },
  "reproductionBuggyBuild": {
    "reproductionDateBeginning": "Oct 6, 2018 10:32:32 PM",
    "reproductionDateEnd": "Oct 6, 2018 11:45:32 PM",
    "totalDuration": 4375,
    "processDurations": {
      "cloning": {
        "nbSteps": 1,
        "totalDuration": 6,
        "stepNames": [
          "CloneRepository"
        ],
        "stepDurations": [
          6
        ]
      },
      "building": {
        "nbSteps": 2,
        "totalDuration": 223,
        "stepNames": [
          "CheckoutBuggyBuild",
          "BuildProject"
        ],
        "stepDurations": [
          0,
          223
        ]
      },
      "testing": {
        "nbSteps": 1,
        "totalDuration": 2337,
        "stepNames": [
          "TestProject"
        ],
        "stepDurations": [
          2337
        ]
      }
    },
    "projectRootPomPath": "./workspace/hekate-io/hekate/438091780/pom.xml"
  },
  "patchDiff": {
    "files": {
      "numberAdded": 0,
      "numberChanged": 0,
      "numberDeleted": 0
    },
    "lines": {
      "numberAdded": 0,
      "numberDeleted": 0
    }
  },
  "tests": {
    "failingModule": "/root/workspace/hekate-io/hekate/438091780/hekate-core",
    "overallMetrics": {
      "numberRunning": 9556,
      "numberPassing": 9503,
      "numberFailing": 1,
      "numberErroring": 52,
      "numberSkipping": 64,
      "failures": [
        {
          "failureName": "com.mysql.jdbc.exceptions.jdbc4.CommunicationsException",
          "isError": true,
          "occurrences": 26
        },
        {
          "failureName": "org.postgresql.util.PSQLException",
          "isError": true,
          "occurrences": 26
        },
        {
          "failureName": "java.lang.AssertionError",
          "isError": false,
          "occurrences": 1
        }
      ]
    },
    "failingClasses": [
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "numberRunning": 39,
        "numberPassing": 13,
        "numberFailing": 0,
        "numberErroring": 26,
        "numberSkipping": 0
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "numberRunning": 39,
        "numberPassing": 13,
        "numberFailing": 0,
        "numberErroring": 26,
        "numberSkipping": 0
      },
      {
        "testClass": "io.hekate.cluster.split.JdbcConnectivityDetectorTest",
        "numberRunning": 6,
        "numberPassing": 5,
        "numberFailing": 1,
        "numberErroring": 0,
        "numberSkipping": 0
      }
    ],
    "failureDetails": [
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testRegisterUnregisterWrongCluster[2:Jdbc[url\u003djdbc:postgresql://127.0.01:5433/hekate-test]]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testToString[2:Jdbc[url\u003djdbc:postgresql://127.0.01:5433/hekate-test]]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testMultipleStopDiscovery[2:Jdbc[url\u003djdbc:postgresql://127.0.01:5433/hekate-test]]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testDiscovery[1:Jdbc[url\u003djdbc:mysql://127.0.0.1:3307/hekate-test?useSSL\u003dfalse]]",
        "failureName": "com.mysql.jdbc.exceptions.jdbc4.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testUnregisterUnknown[2:Jdbc[url\u003djdbc:postgresql://127.0.01:5433/hekate-test]]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testToString[1:Jdbc[url\u003djdbc:mysql://127.0.0.1:3307/hekate-test?useSSL\u003dfalse]]",
        "failureName": "com.mysql.jdbc.exceptions.jdbc4.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testCleanupWithNodes[1:Jdbc[url\u003djdbc:mysql://127.0.0.1:3307/hekate-test?useSSL\u003dfalse]]",
        "failureName": "com.mysql.jdbc.exceptions.jdbc4.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testCleanupWithNodes[1:Jdbc[url\u003djdbc:mysql://127.0.0.1:3307/hekate-test?useSSL\u003dfalse]]",
        "failureName": "com.mysql.jdbc.exceptions.jdbc4.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testRegisterUnregister[1:Jdbc[url\u003djdbc:mysql://127.0.0.1:3307/hekate-test?useSSL\u003dfalse]]",
        "failureName": "com.mysql.jdbc.exceptions.jdbc4.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testDbErrorOnStartDiscovery[1:Jdbc[url\u003djdbc:mysql://127.0.0.1:3307/hekate-test?useSSL\u003dfalse]]",
        "failureName": "com.mysql.jdbc.exceptions.jdbc4.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testDbErrorOnStopDiscovery[2:Jdbc[url\u003djdbc:postgresql://127.0.01:5433/hekate-test]]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testDbErrorOnStopDiscovery[1:Jdbc[url\u003djdbc:mysql://127.0.0.1:3307/hekate-test?useSSL\u003dfalse]]",
        "failureName": "com.mysql.jdbc.exceptions.jdbc4.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testStartStopDiscovery[1:Jdbc[url\u003djdbc:mysql://127.0.0.1:3307/hekate-test?useSSL\u003dfalse]]",
        "failureName": "com.mysql.jdbc.exceptions.jdbc4.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testMultipleStopDiscovery[1:Jdbc[url\u003djdbc:mysql://127.0.0.1:3307/hekate-test?useSSL\u003dfalse]]",
        "failureName": "com.mysql.jdbc.exceptions.jdbc4.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testRegisterUnregisterWrongCluster[1:Jdbc[url\u003djdbc:mysql://127.0.0.1:3307/hekate-test?useSSL\u003dfalse]]",
        "failureName": "com.mysql.jdbc.exceptions.jdbc4.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testDbErrorOnStartDiscovery[2:Jdbc[url\u003djdbc:postgresql://127.0.01:5433/hekate-test]]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testRegisterDuplicate[1:Jdbc[url\u003djdbc:mysql://127.0.0.1:3307/hekate-test?useSSL\u003dfalse]]",
        "failureName": "com.mysql.jdbc.exceptions.jdbc4.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testGetCleanupInterval[1:Jdbc[url\u003djdbc:mysql://127.0.0.1:3307/hekate-test?useSSL\u003dfalse]]",
        "failureName": "com.mysql.jdbc.exceptions.jdbc4.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.split.JdbcConnectivityDetectorTest",
        "testMethod": "testRealDataSources",
        "failureName": "java.lang.AssertionError",
        "detail": "Jdbc[url\u003djdbc:mysql://127.0.0.1:3307/hekate-test?useSSL\u003dfalse]",
        "isError": false
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testGetCleanupInterval[2:Jdbc[url\u003djdbc:postgresql://127.0.01:5433/hekate-test]]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testDiscovery[1:Jdbc[url\u003djdbc:mysql://127.0.0.1:3307/hekate-test?useSSL\u003dfalse]]",
        "failureName": "com.mysql.jdbc.exceptions.jdbc4.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testRegisterUnregister[1:Jdbc[url\u003djdbc:mysql://127.0.0.1:3307/hekate-test?useSSL\u003dfalse]]",
        "failureName": "com.mysql.jdbc.exceptions.jdbc4.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testRegisterDuplicate[2:Jdbc[url\u003djdbc:postgresql://127.0.01:5433/hekate-test]]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testRegisterDuplicate[2:Jdbc[url\u003djdbc:postgresql://127.0.01:5433/hekate-test]]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testMultipleStopDiscovery[2:Jdbc[url\u003djdbc:postgresql://127.0.01:5433/hekate-test]]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testDbErrorOnStopDiscovery[1:Jdbc[url\u003djdbc:mysql://127.0.0.1:3307/hekate-test?useSSL\u003dfalse]]",
        "failureName": "com.mysql.jdbc.exceptions.jdbc4.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testDbErrorOnStartDiscovery[2:Jdbc[url\u003djdbc:postgresql://127.0.01:5433/hekate-test]]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testMultipleStopDiscovery[1:Jdbc[url\u003djdbc:mysql://127.0.0.1:3307/hekate-test?useSSL\u003dfalse]]",
        "failureName": "com.mysql.jdbc.exceptions.jdbc4.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testRegisterUnregisterWrongCluster[2:Jdbc[url\u003djdbc:postgresql://127.0.01:5433/hekate-test]]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testDbErrorOnStopDiscovery[2:Jdbc[url\u003djdbc:postgresql://127.0.01:5433/hekate-test]]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testRegisterDuplicate[1:Jdbc[url\u003djdbc:mysql://127.0.0.1:3307/hekate-test?useSSL\u003dfalse]]",
        "failureName": "com.mysql.jdbc.exceptions.jdbc4.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testGetCleanupInterval[2:Jdbc[url\u003djdbc:postgresql://127.0.01:5433/hekate-test]]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testToString[1:Jdbc[url\u003djdbc:mysql://127.0.0.1:3307/hekate-test?useSSL\u003dfalse]]",
        "failureName": "com.mysql.jdbc.exceptions.jdbc4.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testStartStopDiscovery[2:Jdbc[url\u003djdbc:postgresql://127.0.01:5433/hekate-test]]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testRegisterUnregister[2:Jdbc[url\u003djdbc:postgresql://127.0.01:5433/hekate-test]]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testUnregisterUnknown[1:Jdbc[url\u003djdbc:mysql://127.0.0.1:3307/hekate-test?useSSL\u003dfalse]]",
        "failureName": "com.mysql.jdbc.exceptions.jdbc4.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testUnregisterUnknown[1:Jdbc[url\u003djdbc:mysql://127.0.0.1:3307/hekate-test?useSSL\u003dfalse]]",
        "failureName": "com.mysql.jdbc.exceptions.jdbc4.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testDbErrorOnGetNodes[1:Jdbc[url\u003djdbc:mysql://127.0.0.1:3307/hekate-test?useSSL\u003dfalse]]",
        "failureName": "com.mysql.jdbc.exceptions.jdbc4.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testRegisterUnregisterWrongCluster[1:Jdbc[url\u003djdbc:mysql://127.0.0.1:3307/hekate-test?useSSL\u003dfalse]]",
        "failureName": "com.mysql.jdbc.exceptions.jdbc4.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testStartStopDiscovery[2:Jdbc[url\u003djdbc:postgresql://127.0.01:5433/hekate-test]]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testDbErrorOnStartDiscovery[1:Jdbc[url\u003djdbc:mysql://127.0.0.1:3307/hekate-test?useSSL\u003dfalse]]",
        "failureName": "com.mysql.jdbc.exceptions.jdbc4.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testToString[2:Jdbc[url\u003djdbc:postgresql://127.0.01:5433/hekate-test]]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testStartStopDiscovery[1:Jdbc[url\u003djdbc:mysql://127.0.0.1:3307/hekate-test?useSSL\u003dfalse]]",
        "failureName": "com.mysql.jdbc.exceptions.jdbc4.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testUnregisterUnknown[2:Jdbc[url\u003djdbc:postgresql://127.0.01:5433/hekate-test]]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testDbErrorOnGetNodes[2:Jdbc[url\u003djdbc:postgresql://127.0.01:5433/hekate-test]]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testCleanupWithNodes[2:Jdbc[url\u003djdbc:postgresql://127.0.01:5433/hekate-test]]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testDbErrorOnGetNodes[2:Jdbc[url\u003djdbc:postgresql://127.0.01:5433/hekate-test]]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testGetCleanupInterval[1:Jdbc[url\u003djdbc:mysql://127.0.0.1:3307/hekate-test?useSSL\u003dfalse]]",
        "failureName": "com.mysql.jdbc.exceptions.jdbc4.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testDbErrorOnGetNodes[1:Jdbc[url\u003djdbc:mysql://127.0.0.1:3307/hekate-test?useSSL\u003dfalse]]",
        "failureName": "com.mysql.jdbc.exceptions.jdbc4.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testDiscovery[2:Jdbc[url\u003djdbc:postgresql://127.0.01:5433/hekate-test]]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testRegisterUnregister[2:Jdbc[url\u003djdbc:postgresql://127.0.01:5433/hekate-test]]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testDiscovery[2:Jdbc[url\u003djdbc:postgresql://127.0.01:5433/hekate-test]]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testCleanupWithNodes[2:Jdbc[url\u003djdbc:postgresql://127.0.01:5433/hekate-test]]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      }
    ]
  },
  "commits": {
    "buggyBuild": {
      "repoName": "hekate-io/hekate",
      "branchName": "dev-v.3.x-refactor-messaging",
      "sha": "2a8dcb7698d9f5cd98e1c12eca90e905f8221fb9",
      "url": "http://github.com/hekate-io/hekate/commit/2a8dcb7698d9f5cd98e1c12eca90e905f8221fb9",
      "date": "Oct 6, 2018 9:38:45 PM"
    }
  },
  "projectMetrics": {
    "numberModules": 11,
    "numberSourceFiles": 608,
    "numberTestFiles": 327,
    "numberLibrariesFailingModule": 35
  },
  "type": "only_fail"
}